core/domain/exp_domain.py

```diff
@@ -2331,6 +2331,31 @@ def _convert_states_v29_dict_to_v30_dict(cls, states_dict):
                 del answer_group['tagged_misconception_id']
         return states_dict
 
+    @classmethod
+    def _convert_states_v30_dict_to_v31_dict(cls, states_dict):
+        """Converts from version 30 to 31. Version 31 updates
+        the Voiceover duration to have duration of the audio file
+        under FileModel.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for state_dict in states_dict.values():
+            voiceovers_mapping = (state_dict['recorded_voiceovers']
+                                  ['voiceovers_mapping'])
+            for voiceover in voiceovers_mapping.keys():
```

Comment:
keys are going to be the content Id, right?

Reply:
Yeah it would be that something like 'content or default_outcome'.  I'll introduce a comment clarifying that.

Reply:
I have changed this to use the more precise naming convention to match the 'content_id'.

Reply:
done.